File: dashlane-project/Dashlane/src/main/java/com/dashlane/authentication/accountsmanager/DashlaneAuthenticatorService.java
package com.dashlane.authentication.accountsmanager;

import android.app.Service;
import android.content.Intent;
import android.os.IBinder;


public class DashlaneAuthenticatorService extends Service {
    @Override
    public IBinder onBind(Intent intent) {
        return new DashlaneAuthenticator(this).getIBinder();
    }
}

File: dashlane-project/modules/libraries/account-manager/src/main/kotlin/com/dashlane/authentication/accountsmanager/DashlaneAuthenticatorService.kt
package com.dashlane.authentication.accountsmanager

import android.app.Service
import android.content.Intent
import android.os.IBinder

class DashlaneAuthenticatorService : Service() {
    override fun onBind(intent: Intent): IBinder? {
        return DashlaneAuthenticator(this).iBinder
    }
}